android/src/main/java/com/alinz/parkerdan/shareextension/RealPathUtil.java

```diff
@@ -9,6 +9,171 @@
 import android.content.ContentUris;
 import android.os.Environment;
 
+<<<<<<< HEAD
```

Comment:
@arribbar Still seeing Git conflict commenting in this file. Please remove.

Reply:
Oooh ****, I'll take a look at it this week and get back to you asap. Thank you and sorry for this error